tests/lib.rs

```diff
@@ -21,7 +21,15 @@
 
 extern crate self_encryption;
 extern crate rand;
+extern crate rustc_back;
```

Comment:
Hi Ryman, it says your link is a deprecated library

On 20 March 2015 at 19:35, Ryman <notifications@github.com> wrote:

> In tests/lib.rs
> <https://github.com/dirvine/self_encryption/pull/15#discussion_r26869691>:
>
> > @@ -21,7 +21,15 @@
> >
> >  extern crate self_encryption;
> >  extern crate rand;
> > +extern crate rustc_back;
>
> Those MIT docs are old, you should use the officially hosted ones, tempdir
> is now http://doc.rust-lang.org/std/fs/struct.TempDir.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dirvine/self_encryption/pull/15/files#r26869691>.
>